### DISCUSSION TITLE
Script Not Displaying Any Results......Please Help!

### DISCUSSION THREAD

**Discussion by MikeAlbert4Real** [jef/streetmerchant#1220] [Help]
I installed the script per instructions step by step. Git and node.js installed. Env configured. Script runs, so it seems. When I first start it, I get these messages "INFO: Could not find files for the given pattenrs(s)." Screenshots below. 
![Screenshot1](https://user-images.githubusercontent.com/75456294/101287243-196f4000-37bd-11eb-96f5-0050b5e3d80e.png)
When running even if there is inventory, the script doesnt detect anything. No "instock" or turn green. I included screenshots of my streetmerchant folder, maybe something is missing? 
![Screenshot2](https://user-images.githubusercontent.com/75456294/101287299-71a64200-37bd-11eb-9501-741d7de5e985.png)
and also here is my ..env file. Hoping someone can help as I am lost as to why the script isn't picking up any positives. Thanks in advance.
IN_STOCK_WAIT_TIME="180" 
MAX_PRICE_SERIES_RX6800=""
MAX_PRICE_SERIES_RX6800XT=""
MAX_PRICE_SERIES_RX6900XT=""
MAX_PRICE_SERIES_RYZEN5600=""
MAX_PRICE_SERIES_RYZEN5800=""
MAX_PRICE_SERIES_RYZEN5900=""
MAX_PRICE_SERIES_RYZEN5950=""
MAX_PRICE_SERIES_SONYPS5C="1000"
MAX_PRICE_SERIES_SONYPS5DE="1000"
MICROCENTER_LOCATION=""
MQTT_BROKER_ADDRESS=""
MQTT_BROKER_PORT=""
MQTT_CLIENT_ID=""
MQTT_PASSWORD=""
MQTT_QOS=""
MQTT_TOPIC=""
MQTT_USERNAME=""
NVIDIA_ADD_TO_CART_ATTEMPTS=""
NVIDIA_SESSION_TTL=""
OPEN_BROWSER="true"
PAGE_BACKOFF_MIN=""
PAGE_BACKOFF_MAX=""
PAGE_SLEEP_MIN=""
PAGE_SLEEP_MAX=""
PAGE_TIMEOUT=""
PAGERDUTY_INTEGRATION_KEY=""
PAGERDUTY_SEVERITY=""
PHILIPS_HUE_API_KEY=""
PHILIPS_HUE_CLOUD_ACCESS_TOKEN=""
PHILIPS_HUE_CLOUD_CLIENT_ID=""
PHILIPS_HUE_CLOUD_CLIENT_SECRET=""
PHILIPS_HUE_CLOUD_REFRESH_TOKEN=""
PHILIPS_HUE_LAN_BRIDGE_IP=""
PHILIPS_HUE_LIGHT_COLOR=""
PHILIPS_HUE_LIGHT_IDS=""
PHILIPS_HUE_LIGHT_PATTERN=""
PHONE_CARRIER=""
PHONE_NUMBER=""
PLAY_SOUND="wakeup.wav"
PROXY_ADDRESS=""
PROXY_PROTOCOL=""
PROXY_PORT=""
PUSHBULLET=""
PUSHOVER_TOKEN=""
PUSHOVER_USER=""
PUSHOVER_PRIORITY=""
SCREENSHOT=""
SHOW_ONLY_BRANDS=""
SHOW_ONLY_MODELS=""
SHOW_ONLY_SERIES="sonyps5c,sonyps5de,xboxss,xboxsx"
SLACK_CHANNEL=""
SLACK_TOKEN=""
SMTP_ADDRESS=""
SMTP_PORT=""
STORES="amazon,bestbuy,target,walmart,officedepot,antonline,playstation"
TELEGRAM_ACCESS_TOKEN=""
TELEGRAM_CHAT_ID=""
TWILIO_ACCOUNT_SID=""
TWILIO_AUTH_TOKEN=""
TWILIO_FROM_NUMBER=""
TWILIO_TO_NUMBER=""
TWITCH_ACCESS_TOKEN=""
TWITCH_CHANNEL=""
TWITCH_CLIENT_ID=""
TWITCH_CLIENT_SECRET=""
TWITCH_REFRESH_TOKEN=""
TWITTER_ACCESS_TOKEN_KEY=""
TWITTER_ACCESS_TOKEN_SECRET=""
TWITTER_CONSUMER_KEY=""
TWITTER_CONSUMER_SECRET=""
TWITTER_TWEET_TAGS=""
USER_AGENT=""
WEB_PORT=""

**Comment by jef** [comment]
Consider removing your `.env` file to see if it runs. Ignore the `could not find` part, doesn't effect program (but need to fix).

Let me know how that runs for you.
